HtmlBuilderCore/Elements/HtmlTable.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HtmlBuilderCore.Elements
+{
+    public class HtmlTable : IHtmlElement
+    {
+
```

Comment:
Add more semantic and relevant names for variables and properties :  list  =>should probably be rows. They **represent rows** so name it `rows`. getTable is also sort of _wrong_ it should be probably named row **(singular)**.